vllm/worker/model_runner.py

```diff
@@ -1370,6 +1375,7 @@ def execute_model(
             **MultiModalInputs.as_kwargs(multi_modal_kwargs,
                                          device=self.device),
             **seqlen_agnostic_kwargs)
+        model_forward_end.record()
```

Comment:
can you just not record when the feature is not enabled as well?

Reply:
I measured the time, and it seems to take 100~200 us to record (since we want to make e2e control plane overhead very small, 100~200us seems pretty big for unnecessary work).

Reply:
Done.